va/validation-authority.go

```diff
@@ -333,6 +333,30 @@ func (va *ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdenti
 		return challenge, err
 	}
 
+	contentTypes, ok := httpResponse.Header[http.CanonicalHeaderKey("content-type")]
```

Comment:
This one's my fault: I said in a comment earlier to use `CanonicalHeaderKey` because the docs mentioned that function as the way to get multiple keys. Clearly I wasn't reading closely enough. I support using the staticstring "content-Type" to index here.